lib/netsuite/records/markup_item.rb

```diff
@@ -0,0 +1,43 @@
+module NetSuite
+  module Records
+    class OtherChargeResaleItem
```

Comment:
This needs to be renamed to `MarkupItem`

Reply:
lmaoooooo sorry